safe/gui/tools/geonode_uploader.py

```diff
@@ -165,9 +166,16 @@ def accept(self):
 
         geonode_session = login_user(url, login, password)
 
-        result = upload(url, geonode_session, layer.source())
-        self.button_box.setEnabled(True)
-        disable_busy_cursor()
+        try:
+            result = upload(url, geonode_session, layer.source())
+        except GeoNodeLayerUploadError as e:
+            result = {
+                'success': False,
+                'message': e.message
```

Comment:
`message` is deprecated in Python.
We have warnings in QGIS when InaSAFE is using this `message` member.
Check this PR in InaSAFE: https://github.com/inasafe/inasafe/pull/4558 and this https://stackoverflow.com/questions/1272138/baseexception-message-deprecated-in-python-2-6